packages/flutter/test/widgets/navigator_test.dart

```diff
@@ -4289,14 +4289,14 @@ void main() {
     }
 
     await tester.pumpWidget(build());
-    observer._checkInvocations(<Symbol>[#navigator, #didPush]);
+    observer._checkInvocations(<Symbol>[#navigator, #didPush, #didChangeTop]);
```

Comment:
The `#navigator` look up are originally called from assert in NavigatorState.dispose, and the assert is no longer valid after the update.